deepomatic/api/http_helper.py

```diff
@@ -209,15 +209,29 @@ def recursive_json_dump(prefix, obj, data_dict, omit_dot=False):
         return new_dict
 
     def send_request(self, requests_callable, *args, **kwargs):
-        # requests_callable must be a method from the requests module
```

Comment:
nit: why remove this comment? is it wrong?

Reply:
https://github.com/Deepomatic/deepomatic-client-python/pull/94/files#diff-9166dab22a05c6de78c6aded95aa4d9c7a2596d2d6b1c83523f03566c5154ba4R228